#30: dark background for whole page when dark theme selected

syntagmatic opened the pull request:
Fix for #30, using the theme's `background` and `title.color`, if they are set, for the global body styles on the example page.

domoritz commented:
Thank you for the PR. Can you run `yarn format` once? Also, switching to default crashes the page. 

<img width="553" alt="Screen Shot 2019-07-16 at 10 34 27 AM" src="https://user-images.githubusercontent.com/589034/61278921-652edd00-a7b5-11e9-8331-54dd29707664.png">

syntagmatic commented:
Okay I reworked the `setBackground` function to handle no theme, and removed the `else` clauses. Now it always resets the background/color to `null` before applying theme colors, if they're available.

domoritz commented:
Thank you
